### PR TITLE
Aave Lido Market LTV0

### PR DESCRIPTION
**Pull request by marczeller**

### CODE REVIEW

File: src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+1. Refocusing the instance on its primary use case of leveraged staking with wstETH.
+2. Reducing potential risks associated with using wETH as collateral in this specialized instance.
+3. Encouraging users to migrate towards wstETH for collateral, aligning with the Lido ecosystem focus.
+
+## Specification
+
+This AIP proposes the following changes to the Lido Aave v3 instance for wETH:
+
+- Set wETH LTV (Loan-to-Value) from 82% to 0%
+- Set wETH Liquidation Threshold from 83% to 0%
```
Comment (eboadom) [inline src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md:28]: No, LT can't be reduced to 0

File: src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+
+All other parameters for wETH will remain unchanged:
+
+| Parameter             | Current Value | Proposed Value |
+| --------------------- | ------------- | -------------- |
+| LTV                   | 82%           | 0%             |
+| Liquidation Threshold | 83%           | 0%             |
```
Comment (eboadom) [inline src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md:35]: Same, LT can't be changed to 0

File: src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+| Collateral Enabled    | Yes           | No             |
+| Supply Cap            | 900K          | 900K           |
+| Borrow Cap            | 810K          | 810K           |
+
+## References
+
+- Implementation: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240729_AaveV3Ethereum_WETHLTV0AaveV3LidoInstance/AaveV3Ethereum_WETHLTV0AaveV3LidoInstance_20240729.sol)
+- Tests: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240729_AaveV3Ethereum_WETHLTV0AaveV3LidoInstance/AaveV3Ethereum_WETHLTV0AaveV3LidoInstance_20240729.t.sol)
+- [Snapshot](TODO)
+- [Discussion](TODO)
```
Comment (eboadom) [inline src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md:50]:
```suggestion
- [Discussion](https://governance.aave.com/t/arfc-deploy-a-lido-aave-v3-instance/18047/18)
```

File: src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+| Borrowable            | Yes           | Yes            |
+| Collateral Enabled    | Yes           | No             |
+| Supply Cap            | 900K          | 900K           |
+| Borrow Cap            | 810K          | 810K           |
+
+## References
+
+- Implementation: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240729_AaveV3Ethereum_WETHLTV0AaveV3LidoInstance/AaveV3Ethereum_WETHLTV0AaveV3LidoInstance_20240729.sol)
+- Tests: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240729_AaveV3Ethereum_WETHLTV0AaveV3LidoInstance/AaveV3Ethereum_WETHLTV0AaveV3LidoInstance_20240729.t.sol)
+- [Snapshot](TODO)
```
Comment (eboadom) [inline src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md:49]: Can be removed

**Review by eboadom — APPROVED** [review]

File: src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+    // create proposal
+    vm.startBroadcast();
+    GovV3Helpers.createProposal(
+      vm,
+      payloads,
+      GovernanceV3Ethereum.VOTING_PORTAL_ETH_POL,
+      GovV3Helpers.ipfsHashFile(
+        vm,
+        'src/20240729_AaveV3Ethereum_WETHLTV0AaveV3LidoInstance/WETHLTV0AaveV3LidoInstance.md'
```
Comment (brotherlymite) [inline src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol:55]:
```suggestion
        'src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md'
```

File: src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+
+All other parameters for wETH will remain unchanged:
+
+| Parameter | Current Value | Proposed Value |
+| --------- | ------------- | -------------- |
+| LTV       | 82%           | 0%             |
+
+## References
+
+- Implementation: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240729_AaveV3Ethereum_WETHLTV0AaveV3LidoInstance/AaveV3Ethereum_WETHLTV0AaveV3LidoInstance_20240729.sol)
```
Comment (brotherlymite) [inline src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md:35]:
```suggestion
- Implementation: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.sol)
```

File: src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+- Tests: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240729_AaveV3Ethereum_WETHLTV0AaveV3LidoInstance/AaveV3Ethereum_WETHLTV0AaveV3LidoInstance_20240729.t.sol)
```
Comment (brotherlymite) [inline src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/WETHLTV0AaveV3LidoInstance.md:36]:
```suggestion
- Tests: [AaveV3Ethereum](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.t.sol)
```

**Review by brotherlymite — DISMISSED** [review]

File: src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol [hunk 8]
```diff
@@ -0,0 +1,59 @@
...
+    actions[0] = GovV3Helpers.buildAction(payload0);
+
+    // register action at payloadsController
+    GovV3Helpers.createPayload(actions);
+  }
+}
+
+/**
+ * @dev Create Proposal
+ * command: make deploy-ledger contract=src/20240729_AaveV3Ethereum_WETHLTV0AaveV3LidoInstance/WETHLTV0AaveV3LidoInstance_20240729.s.sol:CreateProposal chain=mainnet
```
Comment (brotherlymite) [inline src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol:32]:
```suggestion
 * command: make deploy-ledger contract=src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol:CreateProposal chain=mainnet
```

File: src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol [hunk 9]
```diff
@@ -0,0 +1,59 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovV3Helpers, IPayloadsControllerCore, PayloadsControllerUtils} from 'aave-helpers/GovV3Helpers.sol';
+import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {EthereumScript} from 'solidity-utils/contracts/utils/ScriptUtils.sol';
+import {AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729} from './AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.sol';
+/**
+ * @dev Deploy Ethereum
+ * deploy-command: make deploy-ledger contract=src/20240729_AaveV3Ethereum_WETHLTV0AaveV3LidoInstance/WETHLTV0AaveV3LidoInstance_20240729.s.sol:DeployEthereum chain=mainnet
```
Comment (brotherlymite) [inline src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol:10]:
```suggestion
 * deploy-command: make deploy-ledger contract=src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol:DeployEthereum chain=mainnet
```

File: src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol [hunk 10]
```diff
@@ -0,0 +1,59 @@
...
+ * verify-command: FOUNDRY_PROFILE=mainnet npx catapulta-verify -b broadcast/WETHLTV0AaveV3LidoInstance_20240729.s.sol/1/run-latest.json
```
Comment (brotherlymite) [inline src/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol:11]:
```suggestion
 * verify-command: FOUNDRY_PROFILE=mainnet npx catapulta-verify -b broadcast/AaveV3EthereumLido_WETHLTV0AaveV3LidoInstance_20240729.s.sol/1/run-latest.json
```

**Review by eboadom — APPROVED** [review]

**Review by brotherlymite — APPROVED** [review]

**Pull request merged**